### PR TITLE
Update to golang 1.11.4

### PR DESCRIPTION
**Pull request by ph**
```
go1.11.4 (released 2018/12/14) includes fixes to cgo, the compiler, linker, runtime, documentation, go command, and the net/http and go/types packages. It includes a fix to a bug introduced in Go 1.11.3 that broke go get for import path patterns containing "...". See the Go 1.11.4 milestone on our issue tracker for details.
```

more details at : https://github.com/golang/go/issues?q=milestone%3AGo1.11.4+label%3ACherryPickApproved

### CODE REVIEW

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -56,6 +55,8 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha1...master[Check the HEAD d
 - When collecting swap metrics for beats telemetry or system metricbeat module handle cases of free swap being bigger than total swap by assuming no swap is being used. {issue}6271[6271] {pull}9383[9383]
 - Adding logging traces at debug level when the pipeline client receives the following events: onFilteredOut, onDroppedOnPublish. {pull}9016[9016]
 - Ignore non index fields in default_field for Elasticsearch. {pull}9549[9549]
+- Update to Golang 1.11.3. {pull}9560[9560]
```
Comment (ruflin) [inline CHANGELOG.asciidoc]: I think the first one becomes obsolete?

Comment (ph) [reply CHANGELOG.asciidoc]: Well, a changelog is a trace of what was done so I am not sure I would remove that.

Comment (ruflin) [reply CHANGELOG.asciidoc]: I would argue this changelog here is for the users and it only matters what happened between releases. If we introduce a bug and fix it before it is released, we also don't list it here.

Comment (ph) [reply CHANGELOG.asciidoc]: I've made the change.

**Comment by ph** [comment]
Packetbeat CI issue is not related to this PR.

**Pull request merged**